src/main/kotlin/uk/gov/gdx/datashare/config/R2dbcConfig.kt

```diff
@@ -0,0 +1,72 @@
+package uk.gov.gdx.datashare.config
```

Comment:
The contents of this file were based off the advice in this issue: https://github.com/pgjdbc/r2dbc-postgresql/issues/529

Reply:
Should you add a comment to that effect?